workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx

```diff
@@ -125,50 +125,47 @@ export function ProvideAppBuildPage({
 
 		let newCategories: Categories[] = [];
 
-		if (appType.value === AppType.CLOUD) {
-			let marketplaceLiferayPlatformOfferingId = 0;
-			let marketplaceLiferayVersionId = 0;
-			let marketplaceEditionId = 0;
-
-			vocabulariesResponse.items.forEach(
-				(vocab: {id: number; name: string}) => {
-					if (
-						vocab.name === 'Marketplace Liferay Platform Offering'
-					) {
-						marketplaceLiferayPlatformOfferingId = vocab.id;
-					}
+		let marketplaceLiferayPlatformOfferingId = 0;
+		let marketplaceLiferayVersionId = 0;
+		let marketplaceEditionId = 0;
+
+		vocabulariesResponse.items.forEach(
+			(vocab: {id: number; name: string}) => {
+				if (vocab.name === 'Marketplace Liferay Platform Offering') {
+					marketplaceLiferayPlatformOfferingId = vocab.id;
+				}
 
-					if (vocab.name === 'Marketplace Liferay Version') {
-						marketplaceLiferayVersionId = vocab.id;
-					}
+				if (vocab.name === 'Marketplace Liferay Version') {
+					marketplaceLiferayVersionId = vocab.id;
+				}
 
-					if (vocab.name === 'Marketplace Edition') {
-						marketplaceEditionId = vocab.id;
-					}
+				if (vocab.name === 'Marketplace Edition') {
+					marketplaceEditionId = vocab.id;
 				}
-			);
+			}
+		);
 
-			const platformOfferingList = await getCategories({
-				vocabId: marketplaceLiferayPlatformOfferingId,
-			});
+		const platformOfferingList = await getCategories({
+			vocabId: marketplaceLiferayPlatformOfferingId,
+		});
 
-			const fullyManagedOption = platformOfferingList.filter(
-				(platformOffering) =>
-					selectedCheckboxValue.includes(platformOffering.name)
-			);
+		const fullyManagedOption = platformOfferingList.filter(
+			(platformOffering) =>
+				selectedCheckboxValue.includes(platformOffering.name)
+		);
 
-			if (fullyManagedOption) {
-				fullyManagedOption.map((managedOption) => {
-					newCategories.push({
-						externalReferenceCode:
-							managedOption?.externalReferenceCode,
-						id: managedOption.id,
-						name: managedOption.name,
-						vocabulary: 'Marketplace Liferay Platform Offering',
-					});
+		if (fullyManagedOption) {
+			fullyManagedOption.map((managedOption) => {
+				newCategories.push({
+					externalReferenceCode: managedOption?.externalReferenceCode,
+					id: managedOption.id,
+					name: managedOption.name,
+					vocabulary: 'Marketplace Liferay Platform Offering',
```

Comment:
Could be 'Marketplace Liferay Platform Offering' used as a constant value?

Reply:
Done!